sql/parser/eval.go

```diff
@@ -259,6 +263,156 @@ func (d DString) String() string {
 	return StrVal(d).String()
 }
 
+// maxTime and minTime are not the theoretical max/min for the time.Time
```

Comment:
If someone does encode the largest time. I suppose we should enforce this on the input side as well right? I suppose if someone wants to put a dummy MAX row after the largest row they would need to use the largest time. In that case we should document it for them to use and we should check inputs against such values.